File: src/ch1/fizzbuzz.py
# 파이썬으로 FizzBuzz 문제 풀기
# 1에서 100까지 반복 --- (*1)
for i in range(1, 101):
    # 조건과 일치하는지 확인 --- (*2)
    if i % 3 == 0 and i % 5 == 0:
        print("FizzBuzz")
    elif i % 3 == 0:
        print("Fizz")
    elif i % 5 == 0:
        print("Buzz")
    else:
        print(i)

File: src/ch1/fizzbuzz.rs
// 러스트로FizzBuzz 문제 풀기
fn main() {
    // 1에서 100까지 반복 --- (*1)
    for i in 1..101 {
        // 조건과 일치하는지 확인 --- (*2)
        if i % 3 == 0 && i % 5 == 0 {
            println!("FizzBuzz");
        } else if i % 3 == 0 {
            println!("Fizz");
        } else if i % 5 == 0 {
            println!("Buzz");
        } else {
            println!("{}", i);
        }
    }
}

